src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import './card';
+@import './stars';
+@import './moto_styles';
+@import './catalog';
+@import './utils/variables';
+
+* {
+  box-sizing: border-box;
+  transition-duration: 300ms;
```

Comment:
It's better to specify to what properties you need to apply the transition.

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/e1ee2234-d5f1-41ac-a8ff-571e53273130)